src/utils/google.js

```diff
@@ -0,0 +1,48 @@
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oidc');
+const { User, Volunteer } = require('../models/user');
+
+passport.use(
+  new GoogleStrategy(
+    {
+      clientID: process.env.GAPP_CLIENT,
+      clientSecret: process.env.GAPP_SECRET,
+      callbackURL: 'http://localhost:8000/auth/google/callback',
```

Comment:
Can you set the callback URL using an env variable? So that we can modify this for production deployment in the future.